### PR TITLE
Issue278 - Fixed top end data values missing when selecting intervals colormap type

### PR DESCRIPTION
**Pull request by lstillwe**
Fixed top end data values missing when selecting intervals colormap type
Also addressed issues with upper and lower bound sliders

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "@mui/joy": "^5.0.0-beta.48",
     "@mui/material": "^6.1.1",
     "@mui/x-date-pickers": "^7.18.0",
-    "@renci/apsviz-geostyler": "15.0.18",
+    "@renci/apsviz-geostyler": "15.0.21",
```
Comment (PhillipsOwen) [inline package.json:58]: there are 2 low severity reported in this update

node_modules/cookie
  express  3.0.0-alpha1 - 4.21.0 || 5.0.0-alpha.1 - 5.0.0
  Depends on vulnerable versions of cookie
  node_modules/express

File: src/components/legend/legend.js [hunk 2]
```diff
@@ -101,12 +101,16 @@ export const MapLegend = () => {
     const nodeRef = useRef(null);
 
     // declare the width min/max for the legend card
-    const minWidth = 45;
-    const maxWidth = 75;
+    //const minWidth = 45;
+    //const maxWidth = 75;
```
Comment (PhillipsOwen) [inline src/components/legend/legend.js:105]: not sure why you just commented these out if you are replacing them.

File: src/components/legend/legend.js [hunk 3]
```diff
@@ -101,12 +101,16 @@ export const MapLegend = () => {
...
+    const minWidth = 65;
+    const maxWidth = 95;
 
     // declare the height min/max for the legend card
-    const minHeight = 250;
-    const maxHeight = 350;
+    //const minHeight = 250;
+    //const maxHeight = 350;
+    const minHeight = 300;
+    const maxHeight = 400;
```
Comment (PhillipsOwen) [inline src/components/legend/legend.js:113]: new settings are producing weirdness. could have been there before tho.

image

**Review by PhillipsOwen — CHANGES_REQUESTED** [review]
basic slider is allowing me to go to 0 with zero range.
image

File: package.json (hunk 1 above)
Comment (lstillwe) [reply package.json:58]: Okay - I'll take a look, might be in a package I can't update.

File: src/components/legend/legend.js (hunk 2 above)
Comment (lstillwe) [reply src/components/legend/legend.js:105]: Ha! - I guess I put back them back to what they were - was tinkering with this quite a bit, couldn't read the number with some the data ranges. I wish we could auto-size to content.

**Comment by lstillwe** [comment]
Okay will check out all this stuff. Thanks for taking a look.

File: src/components/legend/legend.js (hunk 2 above)
Comment (lstillwe) [reply src/components/legend/legend.js:105]: Actually it looks like this in my code ....
 //const minWidth = 45;
    //const maxWidth = 75;
    const minWidth = 65;
    const maxWidth = 95;

**Comment by lstillwe** [comment]
Delivered some updates - please take another look.

**Review by PhillipsOwen — APPROVED** [review]

**Pull request merged**